testing/src/main/java/org/oppia/android/testing/TestMediaPlayer.kt

```diff
@@ -0,0 +1,9 @@
+package org.oppia.android.testing
+
+import android.content.Context
+import android.media.MediaPlayer
+
+interface TestMediaPlayer {
+
+  fun addMediaInfo(context: Context, testUrl: String, testUrl2: String, mediaPlayer: MediaPlayer)
```

Comment:
Can we be clearer in the naming here than just ``testUrl`` and ``testUrl2``?

Reply:
Done